efinance/stock/getter.py

```diff
@@ -221,8 +237,8 @@ def get_quote_history(
     4759  贵州茅台  600519  2021-07-28  1703.00  1768.90  1788.20  1682.12   85369  1.479247e+10  6.19  3.27  56.01   0.68
     4760  贵州茅台  600519  2021-07-29  1810.01  1749.79  1823.00  1734.34   63864  1.129957e+10  5.01 -1.08 -19.11   0.51
 
-    >>> # 获取多只股票历史行情
-    >>> stock_df = ef.stock.get_quote_history(['600519','300750'])
+    >>> # 获取多只A股股票历史行情
```

Comment:
Keep your new example and the example of obtaining market history without restricting market_type.

Reply:
See [retrieve original example, while illustrate new proposed features](https://github.com/Micro-sheep/efinance/pull/161/commits/6e50963c9dafde40e001131246dc4774b502a95e)